cli/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "msadoc-cli",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "keywords": [
+    "cli",
+    "decentralized",
+    "documentation",
+    "microservice"
+  ],
+  "files": [
+    "dist/"
+  ],
+  "type": "module",
+  "module": "dist/cli.js",
+  "bin": {
+    "msadoc": "dist/cli.js"
+  },
```

Comment:
This is how you define the name of our CLI. If I used something like 

```
"foo": "dist/cli.js"
```
then the user would have to run our CLI using "foo" instead of "msadoc" (i.e. `foo upload <...>` instead of `msadoc upload <...>`)